hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerTestUtilities.java

```diff
@@ -26,16 +26,19 @@
 
 import java.util.Set;
 
-public class CapacitySchedulerTestBase {
-  protected final int GB = 1024;
+public final class CapacitySchedulerTestUtilities {
+  public static final int GB = 1024;
+
+  private CapacitySchedulerTestUtilities() {
+    throw new IllegalStateException("Utility class");
```

Comment:
Perhaps this is a bit of an overkill, it's already private, we don't really need this exception.

Reply:
Removed the exception, and rebased to trunk.